dev.nuculabs.ImageTagger.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=Image Tagger
+Exec=/app/bin/image/bin/ImageTagger
```

Comment:
this should just the the bane if the executable and the executable should be in the path. (via a symlink if needed)